AttackItem.h

```diff
@@ -1 +1,14 @@
-#include "Item.h"
\ No newline at end of file
+#include "Utility.h"
+
+struct AttackItem : public Item
+{
+public:
+    // Constructor
+    AttackItem();
+
+    // Destructor
+    virtual ~AttackItem() override;
+
+    // Pure virtual member function
+    virtual void use(Character*) override;
```

Comment:
You mark functions that you want derived classes to be able to implement as `virtual`.
No classes are inheriting from `AttackItem`.
These functions do not need to be marked `virtual`.
- [ ] revise